client/app/account/billing/payment/method/default/billing-payment-method-default.controller.js

```diff
@@ -0,0 +1,40 @@
+export default class BillingPaymentMethodDefaultCtrl {
+  constructor($injector, $q, $uibModalInstance, paymentMethodToEdit, ovhPaymentMethod) {
+    /* @ngInject */
+
+    // dependencies injections
+    this.$injector = $injector;
+    this.$q = $q;
+    this.$uibModalInstance = $uibModalInstance;
+    this.paymentMethodToEdit = paymentMethodToEdit;
+    this.ovhPaymentMethod = ovhPaymentMethod;
+
+    // other attributes used in view
+    this.loading = {
+      translations: false,
+      save: false,
+    };
+  }
+
+  onPrimaryActionClick() {
```

Comment:
Maybe this should be renamed to be more explicit 
```suggestion
  setAsDefault() {
```